recipe/reset_from_gitlab_artifact.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Deployer;
+use SourceBroker\DeployerExtendedDatabase\Utility\ConsoleUtility as DatabaseConsoleUtility;
+
+task('reset:from_gitlab_artifact', function () {
+    // set in deploy.php as https://gitlab.example.org/api/v4/projects/<project-id>/jobs/artifacts/<branch>/download?job=<job-of-artifact>
+    $url = get('reset_gitlab_artifact_url');
+    // Gitlab API token for the repository where the artifact is stored. Can be a project token.
+    $gitlabApiToken = (new DatabaseConsoleUtility())->getOption('token', true);
+    // Database dumpcode that was used during the creation of the Gitlab artifact.
+    $dumpCode = (new DatabaseConsoleUtility())->getOption('dumpcode', true);
+
+    if ( filter_var($url, FILTER_VALIDATE_URL) && preg_match('#jobs\/artifacts\/.+\/download$#', parse_url($url, PHP_URL_PATH)) ) {
```

Comment:
Please don't use nested conditions. The URL validation can be a standalone condition.,